Makefile

```diff
@@ -33,7 +33,7 @@ JUNIT_REPORT ?= true
 # Example:
 #   make
 #   make all
-#   make all WHAT=cmd/oc GOFLAGS=-v
+#   make all WHAT=cmd/openshift-tests GOFLAGS=-v
```

Comment:
@soltysh i would not use openshift-tests as an example :D but 🤷‍♂

Reply:
well, I'm not sure what's else left here :wink: